netlify/_config.js

```diff
@@ -10,4 +10,9 @@ module.exports = {
   moralisAppId: process.env.GATSBY_MORALIS_APP_ID,
   kickboxApiKey: process.env.KICKBOX_API_KEY,
   apiToken: process.env.C4_API_TOKEN,
+  // keep in sync with START and END in register/bot.tsx
```

Comment:
Could both this i.e. the `register-bot.ts` **and** the `register/bot.tsx` grab this out of the bot-race-qualifier JSON that's now in "the graph" (and also could be imported just as a repo file)?